fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/serialization/LoanApplicationValidator.java

```diff
@@ -2123,6 +2126,7 @@ public static void validateOrThrow(String resource, Consumer<DataValidatorBuilde
         baseDataValidator.accept(dataValidatorBuilder);
 
         if (!dataValidationErrors.isEmpty()) {
+            logger.info(dataValidationErrors.toString());
```

Comment:
Please remove this! Exception handler can decide whether they are logging something or not. Also dont use "INFO" level logging for logging out exceptions!

Reply:
Why was this comment resolved?